python/tvm/ir/json_compact.py

```diff
@@ -84,12 +91,26 @@ def _update_global_key(item, _):
         del item["global_key"]
         return item
 
+    def _update_from_std_str(key):
+        def _convert(item, nodes):
+            str_val = item["attrs"][key]
+            jdata = json.loads(tvm.ir.save_json(tvm.runtime.String(str_val)))
```

Comment:
Should we bring the serialization of tvm:String out of save_json?

Reply:
Can you elaborate a bit about what do you mean? Right now all tvm objects are serialized through save_json, what we are doing here is to get that particular entry and append to the end of the nodes.